README.md

```diff
@@ -83,10 +78,7 @@ mcollective class, with secondary configuration managed by the defined types
 
 ```puppet
 node 'broker1.example.com' {
-  class { '::mcollective':
-    middleware       => true,
-    middleware_hosts => [ 'broker1.example.com' ],
-  }
+  include activemq
```

Comment:
Yes, but should there not be an `include mcollective` as well? Perhaps even passing the `middleware_hosts` parameter (which is still supported)?